vexim/adminaliasaddsubmit.php

```diff
@@ -43,6 +43,11 @@
     die;
   }
 
+  # check_mail_address() will die if the localpart (>64 char) or the total address (>254 char) is too long
```

Comment:
This comment seems unnecessary, especially considering that you haven't added it in other places.